CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # eslint-plugin-import-x
 
+## Unreleased
```

Comment:
These should be deleted, we're using tools to manage CHANGELOG automatically

Reply:
Ah, that's good to know. I was going by @changeset-bot's [comment](https://github.com/un-ts/eslint-plugin-import-x/pull/248#issuecomment-2730371366):

> If these changes should result in a version bump, you need to add a changeset.

The [CONTRIBUTING.md](https://github.com/un-ts/eslint-plugin-import-x/blob/96c08b49c60bd43c984f11809953ed03d0f5781c/CONTRIBUTING.md#changelog) also advises:

> Please add a quick blurb to the [**Unreleased**](./CHANGELOG.md#unreleased) section of the change log.

Reply:
Ah, we definitely should update the document accordingly.